target/linux/sunxi/config-5.4

```diff
@@ -1,24 +1,32 @@
+CONFIG_64BIT_TIME=y
 # CONFIG_AHCI_SUNXI is not set
 CONFIG_ALIGNMENT_TRAP=y
-# CONFIG_ARCH_AXXIA is not set
```

Comment:
It would be nice to get the refresh of configs into separate commits and due to the number of changes in this file (+48 -53) and a53's I would probably try to make the review easier by splitting it into separate commits per subtarget as well.